README.md

```diff
@@ -2,7 +2,7 @@
 
 This repository contains the value sets referenced by the EU Digital COVID Certificate (DCC) [JSON Schema](https://github.com/ehn-dcc-development/ehn-dcc-schema).
 
-Release: 1.3.0
+Release: 2.1.0
```

Comment:
`main` is at release tagged `2.2.0`. I thought the v1.4 update was approved?

Reply:
@dslmeinte The readme is updated and already refers to the latest release `2.2.0`
- https://github.com/ehn-dcc-development/ehn-dcc-valuesets/tree/main
- https://github.com/ehn-dcc-development/ehn-dcc-valuesets/tree/release/2.2.0

This PR is for the `relase/2.1.0` branch where we missed to mark it in the readme
- https://github.com/ehn-dcc-development/ehn-dcc-valuesets/tree/release/2.1.0

![image](https://user-images.githubusercontent.com/690575/140897279-a002a0aa-dd57-453b-92e1-bc8d01d3e45c.png)

Reply:
Uh, that's a clear sign I need another coffee!

Reply:
But more seriously: this commit wasn't signed, which blocks merging.

Reply:
@dslmeinte I reworked and signed the commit.